gandalf.js

```diff
@@ -2,57 +2,58 @@
  * Extras Plugin
```

Comment:
SCRIPT-This block of code could benefit from some comments explaining the logic behind it.